packages/x-components/src/x-modules/tagging/wiring.ts

```diff
@@ -219,6 +216,23 @@ export function createTrackDisplayWire(property: keyof Tagging): Wire<Taggable>
   );
 }
 
+/**
+ * Factory helper to create a wire for set the queryTagging.
```

Comment:
This way: 
```suggestion
 * Factory helper to create a wire for setting the queryTagging.
```
Or this one: 
```suggestion
 * Factory helper to create a wire to set the queryTagging.
```